### ISSUE TITLE
How to set Screen from the Kameleo API?

### ISSUE THREAD

**Issue by JonathanR25** [kameleo-io/local-api-client-typescript#15]
What would be the proper code syntax form to set the Screen size for a profile from the Kameleo API?

Also, could your provide some example code?

**Comment by MBailey01** [comment]
Here is an example that you can use to set the Screen for a profile through the API:

```
const createProfileRequest = BuilderForCreateProfile
  .forBaseProfile(chromeBaseProfileList[0].id)
  .setRecommendedDefaults()
  .setScreen('manual', '800x600') \  # Options: ('automatic', 'manual', 'off') # In this example we used the value of "800x600"
  .build();
const profile = await client.createProfile({ body: createProfileRequest });
```

These answers are provided from the Kameleo [Set Screen from API](https://help.kameleo.io/hc/en-us/articles/15081079308060-Set-Screen-from-API) article.

**Comment by KameleoStaff** [comment]
We recently released Kameleo v4.0.0, which introduced several changes and improvements to the platform. As a result, this code has been updated to be fully compatible with the latest version.

```
@type {import('@kameleo/local-api-client').CreateProfileRequest} */
const createProfileRequest = {
  fingerprintId: fingerprints[0].id,
  screen: {
    value: "manual", # Options: ('automatic', 'manual', 'off')
    extra: "800x600",
  },
};

const profile = await client.profile.createProfile(createProfileRequest);
```

These answers are provided from the Kameleo [Set Screen from API](https://help.kameleo.io/hc/en-us/articles/15081079308060-Set-Screen-from-API) article.
